src/tests/stub_libip.cc

```diff
@@ -12,6 +12,67 @@
 #define STUB_API "ip/libip.la"
 #include "tests/STUB.h"
 
+#include "ip/Address.h"
+Ip::Address::Address(const struct in_addr &) STUB
```

Comment:
This and most other additions in this stub file are _not_ from master PR 1297 as this PR branch effectively claims. They are from master commit b709c37 that is _not_ mentioned in the PR branch. However, that commit b709c37 has changes that are not present in this PR branch.

Reply:
ack, redone